Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -358,42 +360,42 @@ cancelRidePopUpConfig state =
         , config = state.data.config
         }
 
-genderBannerConfig :: ST.HomeScreenState -> Banner.Config
-genderBannerConfig state =
+genderBannerConfig :: forall a. ST.HomeScreenState -> a -> BannerCarousel.Config a
+genderBannerConfig state action =
   let
-    config = Banner.config
+    config = BannerCarousel.config action
     config' = config
       {
         backgroundColor = Color.lightMintGreen
       , title = (getString COMPLETE_YOUR_PROFILE_FOR_A_PERSONALISED_RIDE_EXPERIENCE)
       , titleColor = Color.elfGreen
       , actionText = (getString UPDATE_NOW)
       , actionTextColor = Color.elfGreen
-      , imageUrl = fetchImage FF_COMMON_ASSET "ny_ic_banner_gender_feat"
-      , isBanner = state.props.isBanner
+      , imageUrl = "ny_ic_banner_gender_feat"
+      , type = BannerCarousel.Gender
       }
   in config'
 
-disabilityBannerConfig :: ST.HomeScreenState -> Banner.Config
-disabilityBannerConfig state =
+disabilityBannerConfig :: forall a. ST.HomeScreenState -> a -> BannerCarousel.Config a
+disabilityBannerConfig state action =
   let
-    config = Banner.config
+    config = BannerCarousel.config action
     config' = config
       {
         backgroundColor = Color.paleLavender
       , title = (getString NOW_GET_ASSISTED_RIDES)
       , titleColor = Color.purple
       , actionText = (getString UPDATE_PROFILE)
       , actionTextColor = Color.purple
-      , imageUrl = fetchImage FF_ASSET "ny_ic_accessibility_banner_img"
-      , stroke = "1,"<> Color.fadedPurple
+      , imageUrl = "ny_ic_accessibility_banner_img"
```

Comment:
@Vignesh-772 why are we not using fetchImage here.

Reply:
for carousel we need to either name or url we can't give both.